boards/lora-e5-dev/doc.txt

```diff
@@ -1,13 +1,24 @@
 /**
  * @defgroup    boards_lora-e5-dev LoRa-E5 Development Board - STM32WLE5JC
  * @ingroup     boards
- * @brief       Support for the LoRa-E5 Development Board - STM32WLE5JC board.
+ * @brief       Support for the LoRa-E5 Development Board - STM32WLE5JC.
  *
  * @warning     This BOARD comes with arduino style pin headers, but the gpio
  *              mapping does not map to arduino BOARDs, even 3.3V and 5V pins
  *              are placed differently, so don't use arduino expansion-boards
  *              since these might short-circuit the mcu and/or expansion-board.
  *
+ * The LoRa-E5 Dev Board is an evaluation BOARD for the Seeed Studio LoRa-E5 STM32WLE5JC
```

Comment:
```suggestion
 * The LoRa-E5 Dev Board is an evaluation board for the Seeed Studio LoRa-E5 STM32WLE5JC
```

Do you have some sort of auto correction enabled or do you really write _board_ capitalized ?